.gitignore

```diff
@@ -1,2 +1,3 @@
 target
 Cargo.lock
+.DS_Store
```

Comment:
Can we not include these files?

Reply:
yeah let me back this out

Reply:
And the file in the `benches` directory (I can't comment on it directly).